plugins/modules/database/saphana/hana_query.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2021, Rainer Leber <rainerleber@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: hana_query
+
+short_description: Execute SQL on HANA.
+
+version_added: "3.2.0"
+
+description: This module executes SQL statements on HANA with hdbsql.
+
+options:
+    sid:
+        description: The system ID.
+        type: str
+        required: true
+    instance:
+        description: The instance number.
+        type: str
+        required: true
+    user:
+        description: A dedicated username. Defaults to C(SYSTEM).
+        type: str
+        default: SYSTEM
+    password:
+        description: The password to connect to the database.
+        type: str
+        required: true
+    autocommit:
+        description: Autocommit the statement.
+        type: bool
+        default: true
+    host:
+        description: The Host IP address. The port can be defined as well.
+        type: str
+    database:
+        description: Define the database on which to connect.
+        type: str
+    encrypted:
+        description: Use encrypted connection. Defaults to C(false).
+        type: bool
+        default: false
+    filepath:
+        description:
+        - One or more files each containing one SQL query to run.
+        - Must be a string or list containing strings.
+        type: list
+        elements: path
+    query:
+        description:
+        - SQL query to run. Multiple queries can be passed using YAML list syntax.
+        - Must be a string or list containing strings.
+        type: list
+        elements: str
+
+author:
+    - Rainer Leber (@rainerleber)
+'''
+
+EXAMPLES = r'''
+
+- name: Simple select query
+  community.general.hana_query:
+    sid: "hdb"
+    instance: "01"
+    password: "Test123"
+    query: "select user_name from users"
+
+- name: Run several queries
+  community.general.hana_query:
+    sid: "hdb"
+    instance: "01"
+    password: "Test123"
+    query:
+    - "select user_name from users;"
+    - select * from SYSTEM;
+    host: "localhost"
+    autocommit: False
+
+- name: Run several queries from file
+  community.general.hana_query:
+    sid: "hdb"
+    instance: "01"
+    password: "Test123"
+    filepath:
+    - /tmp/HANA_CPU_UtilizationPerCore_2.00.020+.txt
+    - /tmp/HANA.txt
+    host: "localhost"
+
+
+'''
+
+RETURN = r'''
+
+stdout:
+    type: str
+    description: A json string of the returned value from the SQL query.
```

Comment:
```suggestion
    description: A JSON string of the returned values from the SQL queries.
```
I don't think returning a JSON **string** makes sense though. You should return the data in a format that can be directly used.